### ISSUE TITLE
install failed?!

### ISSUE THREAD

**Issue by OneLine-Lee** [dbcli/mycli#565]
![20180123194628](https://user-images.githubusercontent.com/8857021/35274213-39388e2c-0076-11e8-8f29-67e1448070df.png)

**Comment by OneLine-Lee** [comment]
anyone can tell me how to fix it?

**Comment by amjith** [comment]
It looks like you don't have permission to create some folders. You might have to run it with `sudo`. 

Try:
```
sudo pip install mycli
```

**Comment by pensivebrian** [comment]
I agree with @amjith that it's probably a permissions issue, but the presence of "D:\Program Files" in the path means you're probably running on Windows. This makes the instructions a bit different. On Windows, you'll have to run the pip install command from an elevated command prompt. 

To create an elevated command prompt, when you launch cmd.exe, try right clicking on the icon, and select the "Run as administrator". Then, run the `pip install mycli` command in the console.
